### PR TITLE
added twitter and slack link on navbar

### PR DESCRIPTION
**Pull request by fiestared**
@sot528 
- added twitter and slak icon on navbar
- their color is white

please check the points below
- css coding and naming( Whether I correctly write BEM method or not)
- no layout collapse

### CODE REVIEW

File: src/includes/navbar.pug [hunk 1]
```diff
@@ -18,6 +18,11 @@ block links
       li
         a.smooth-anchor(href='#meetus') #{navbar.meetus}
       li
+        a.btn.btn-social-icon.btn-lg.navbar__sns-links__link(href='https://twitter.com/ALIS_media', target='blank')
```
Comment (sot528) [inline src/includes/navbar.pug]: [nits]
Please integrate redundant value into one place.
- `https://twitter.com/ALIS_media`

File: src/includes/navbar.pug [hunk 2]
```diff
@@ -18,6 +18,11 @@ block links
...
+          i.fa.fa-twitter
+        a.btn.btn-social-icon.btn-lg.navbar__sns-links__link(href='https://alis-slack.herokuapp.com/', target='blank')
```
Comment (sot528) [inline src/includes/navbar.pug]: [nits]
Please integrate redundant value into one place.
- https://alis-slack.herokuapp.com

**Comment by sot528** [comment]
[MUST]
Please fix:

**Comment by fiestared** [comment]
@sot528 
I modified the navbar so please check it. I'll integrate redutant parameters in another task, so just check navbar works well

**Comment by sot528** [comment]
[MUST]
Please fix it.

File: src/scss/home.scss [hunk 3]
```diff
@@ -575,6 +583,63 @@ body {
...
+  .navbar-toggle {
+    display: block;
+  }
+
+  .navbar-collapse {
+    border-top: 0;
+  }
+
+  .navbar-collapse.collapse {
+    display: none!important;
```
Comment (sot528) [inline src/scss/home.scss]: [FYI]
Maybe this code is from Bootstarp. 
So no problem in this case.

However, normally `!important` is not so good way to write CSS.

**Pull request merged**